pom.xml

```diff
@@ -271,6 +272,14 @@
 				<artifactId>jackson-databind</artifactId>
 				<version>${jackson-databind.version}</version>
 			</dependency>
+
+			<!-- https://mvnrepository.com/artifact/org.jetbrains/annotations -->
+			<dependency>
+				<groupId>org.jetbrains</groupId>
+				<artifactId>annotations</artifactId>
+				<version>${jetbrains.annotations.version}</version>
+				<scope>compile</scope>
```

Comment:
Can we add `optional` here? I guess, that this dependency is not required at runtime, so it would be save to add an `optional` flag here...

Reply:
I don't know, I only saw it with this configuration. What do you think @jnioche?

Reply:
No idea. it is not really optional since it is needed a compilation time. Am not that bothered either way as long as it compiles

Reply:
Importing projects will need to declare it if optional=true. 

We can leave it as is and importing projects can exclude it - these annotations are mit read at runtime, so should be save but we can test/change in another PR.